src/mono/sample/wasi/Directory.Build.targets

```diff
@@ -15,13 +15,13 @@
       TODOWASI - - tcplisten localhost:64000 - - env DEBUGGER_FD=4
     -->
     <Exec WorkingDirectory="bin/wasi-wasm/AppBundle"
-          Condition="'$(WasmBuildNative)' != 'true'"
+          Condition="'$(WasmBuildNative)' != 'true' and '$(OutputType)' != 'Library'"
```

Comment:
Are these changes needed?

Reply:
These I added for `Wasi.Console.Sample` testing purposes as library. Will revert it.